docs/02-BUILD_THE_RIGHT_THING.md

```diff
@@ -0,0 +1,37 @@
+## Where do i start?
+So you definitely need a design-system, here are some questions to help guide you in `building the right thing`, before you start trying to `build the thing right`.
+
```

Comment:
Reshuffled these numerically to highlight order of importance of these questions. i.e. we should be defining principle sand design decisions first before we start thinking about the kind of artifacts we want.